site/en/docs/extensions/mv3/tut_websockets/index.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: 'layouts/doc-post.njk'
+title: 'Using WebSockets in Service Workers'
+seoTitle: 'How to use WebSockets in Chrome Extension Service Workers'
+date: 2023-07-04
+description: 'Step-by-step instructions on how to connect to a WebSocket in your Chrome Extension.'
+---
+
+This tutorial demonstrates how to connect to a WebSocket in your Chrome Extension's service worker. You can find a [working example on Github](https://github.com/GoogleChrome/chrome-extensions-samples/tree/main/functional-samples/tutorial.websockets).
+
+## Background
```

Comment:
When I read the two first paragraphs, I get the feeling that it could be simplified a bit more. There's a few items that seem a bit redundant. 

Do you think we could tweak it a bit to make it more concise?

Reply:
Removed the second paragraph and rephrased the first one - wdyt?